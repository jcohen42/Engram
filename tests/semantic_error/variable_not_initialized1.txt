Begin program:
	Output "What's up".
	Set x equal to y.
End program.