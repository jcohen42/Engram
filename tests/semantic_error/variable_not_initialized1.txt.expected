Error: Variable y was not initialized before it was used on line 3.
Please ensure that this variable is declared beforehand.